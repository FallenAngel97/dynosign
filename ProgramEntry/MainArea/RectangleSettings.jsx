import React from 'react';
import './RectangleSettings.scss';

/**
 * defines the common figures propertis. Offset, height, width.
 * @module RectangleSettings
 */

export default class RectangleSettings extends React.Component {
  render () {
    return (
      <div id='rectangle_settings'>
        Left: <input />
        Top: <input />
        Height: <input />
        Width: <input />
      </div>
    )
  }
}
