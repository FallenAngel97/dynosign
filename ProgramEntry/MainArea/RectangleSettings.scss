#rectangle_settings {
    input {
        width: 60px;
    }
}