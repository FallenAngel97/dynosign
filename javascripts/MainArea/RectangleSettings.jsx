import React from 'react';
import './RectangleSettings.scss';

export default class RectangleSettings extends React.Component {
  render () {
    return (
      <div id='rectangle_settings'>
        Left: <input />
        Top: <input />
        Height: <input />
        Width: <input />
      </div>
    )
  }
}
