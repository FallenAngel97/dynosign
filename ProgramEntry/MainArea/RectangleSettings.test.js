/* global shallow */
import React from 'react';
import RectangleSettings from './RectangleSettings';

describe('RectangleSettings test', () => {
  it('should render properly', () => {
    const rectangleSettings = shallow(<RectangleSettings />);
    expect(rectangleSettings).toMatchSnapshot();
  });
});
