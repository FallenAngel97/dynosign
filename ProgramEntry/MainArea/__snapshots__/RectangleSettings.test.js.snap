// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RectangleSettings test should render properly 1`] = `ShallowWrapper {}`;
